doc/classes/EditorSettings.xml

```diff
@@ -714,6 +714,9 @@
 		<member name="project_manager/default_renderer" type="String" setter="" getter="">
 			The renderer type that will be checked off by default when creating a new project. Accepted strings are "forward_plus", "mobile" or "gl_compatibility".
 		</member>
+		<member name="project_manager/directory_naming_convention" type="int" setter="" getter="">
+			Directory naming convention for project manager. Options are "None", "kebab-case", "snake_case", "camelCase", "PascalCase", or "Title Case".
```

Comment:
I think None should be explained, as it's less obvious.

Reply:
Does "None" imply that the project title becomes the directory, spaces and all?

"Unchanged" or "Literal" or something describing the behavior may be more immediately apparent.

Reply:
> Does "None" imply that the project title becomes the directory, spaces and all?

Yes, although it also replaces invalid characters.

Reply:
Changed to

> "None" (project name is directory name), "kebab-case" (default), ...

Reply:
I also changed "None" to "No convention"